Makefile

```diff
@@ -64,14 +64,21 @@ ios:
 	@echo "Done building."
 	@echo "Import \"$(GOBIN)/Kusd.framework\" to use the library."
 
-test: all
+test: all mocks
 	cd client; build/env.sh go run build/ci.go test
 
 test_notifications: dep
 	cd notifications && \
 	$(GOPATH)/bin/dep ensure --vendor-only && \
 	go test ./... -tags=integration
 
+mocks:
+	go get github.com/vektra/mockery/.../
```

Comment:
Can you add this to the tools sections using the same pattern to only install it if it's missing?